bostardecksweb/settings.py

```diff
@@ -43,6 +43,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'customMiddleware',
```

Comment:
give it proper name

Reply:
why this isn't proper?

Reply:
I think that creating middleware this kind isn't proper solution this problem, and creating something called custom is wrong. It says nothing about kind of operation